package/MDAnalysis/core/AtomGroup.py

```diff
@@ -4170,10 +4170,10 @@ def __init__(self, *args, **kwargs):
              topology) is always required.
 
           *permissive*
```

Comment:
This keyword is deprecated now. You should mention it in the docs. Don't issue a separate deprecation warning but mention that this kwarg will vanish soon.